source/common/json/config_schemas.cc

```diff
@@ -672,7 +672,9 @@ const std::string Json::Schema::ROUTE_ENTRY_CONFIGURATION_SCHEMA(R"EOF(
       "cluster" : {"type" : "string"},
       "cluster_header" : {"type" : "string"},
       "weighted_clusters": {"$ref" : "#/definitions/weighted_clusters"},
+      "scheme_redirect" : {"type" : "string"},
       "host_redirect" : {"type" : "string"},
+      "port_redirect" : {"type" : "integer"},
       "path_redirect" : {"type" : "string"},
```

Comment:
Changes to this file aren't needed as this is only for v1 and the change is in the v2 API.